index.html

```diff
@@ -34,13 +36,14 @@
         }
 
         #theme-toggle {
-            margin-top: 5px;
-            padding: 10px;
+            margin-top: 3rem;
+            padding: 0.7rem;
             cursor: pointer;
             display: inline-block;
             background-color: rgb(213, 184, 238);
             border: none;
             font-size: 1rem;
+            border-radius: 60%;
```

Comment:
And your icon button is not fully circle. Its looks weird! Could you please fix this? it should be proper circle!

Reply:
Done!